src/main/webapp/css/transition.css

```diff
@@ -0,0 +1,130 @@
+/* Loading Screen Styling */
+
+.loading-screen {
+    position: relative;
+    padding-left: 0;
+    padding-right: 0;
+    padding-top: 0;
+    background-color: #4885ed;
+    width: 100%;
+    height: 100%;
+    transform: scaleY(0) skewX(0);
+    transform-origin: top left;
+}
+
+.loading-container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+    overflow: hidden;
+    z-index: 10;
+    pointer-events: none;
+}
+
+/* Loading Screen Styling Ends */
+
+/* ================= */
+
+/* Heading Container Styling */
```

Comment:
please match this to the format of the other sections

Reply:
done